ChangeLog

```diff
@@ -28,6 +28,10 @@ Release date: TBA
 
   Closes #1737
 
+* Fix false positive with inference of `http` module when iterating `HTTPStatus`.
```

Comment:
Single backticks seems to just [create italics](https://astroid.pycqa.org/en/latest/changelog.html)
```suggestion
* Fix false positive with inference of ``http`` module when iterating ``HTTPStatus``.
```

Reply:
Thanks!